client/lib/purchases/index.js

```diff
@@ -146,15 +147,19 @@ function isPaidWithPaypal( purchase ) {
 }
 
 function isPaidWithIdeal( purchase ) {
-	return 'iDEAL' === purchase.payment.type;
+	return 'ideal' === purchase.payment.type;
```

Comment:
What if we add change this (and the ones below) to  `purchase.payment.type.toLowerCase()`, which means we could deploy this without worrying about case sensitivity in D9161-code ?